_episodes/10-publication.md

```diff
@@ -12,6 +12,136 @@ keypoints:
 - "First key point. Brief Answer to questions. (FIXME)"
 ---
 
+## Publications
+
+<img src="https://the-turing-way.netlify.app/_images/evolution-open-research.jpg" alt="drawing" width="500"/> 
+
+While the output of research projects is usually centred around publishing a journal article, this format of science communication and knowledge sharing is increasingly restrictive with the new ways scientific research is conducted. The requirements from journals themselves is also expanding, you are now often asked to upload data sets and code as part of your publication. Releasing data is increasingly a requirement from funding bodies, and outputs from research groups can go beyond a single paper, releasing tools and methods that can be used worldwide.
+
+In general there are different degrees of openness. 
+
+- Fully private data and code, unavailable
+- Pseudo-open -- "available on request"
+- Released static code parallel to a research paper, see Zenodo or Figshare
+- Open online repository -- CRAN, GitHub
```

Comment:
I don't think we want call Github a repository - maybe call it an open-source project tool

If this section is about publication, we need to be focused on long-term archiving so stress Trusted Repositories such as zenodo, osf, figshare, etc.

Reply:
the question is still open I think, as for recommendation for FAIR software often starts with having it on a git repo system, only mentioning archives later on. I think there is no hierarchy of openes between the two. The important missing parterre  is the license.